Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -17,19 +17,39 @@ private struct TabTrayControllerUX {
     static let TabTitleTextFont = AppConstants.DefaultSmallFont
 }
 
+private struct TabCellUX {
+    // Properties for panning animation
+    static let deleteThreshold = 140
+    static let totalRotationInDegrees = 10.0
+    static let totalScale = CGFloat(0.9)
+    static let totalAlpha = CGFloat(0.7)
+    static let minExitVelocity = 800.0
+    static let recenterAnimationDuration = 0.15
```

Comment:
Can you make all these start with an uppercase letter. That is what we do in the other `*UX` structs too, to note that these things are constants.